cabal.project

```diff
@@ -38,3 +38,18 @@ import: ./asserts.cabal
 if(os(windows))
   constraints:
     bitvec -simd
+
+allow-newer:
+  -- All these cardano-ledger packages have been fixed on master but not
+  -- yet released to CHaP.
+  , cardano-ledger-alonzo:plutus-ledger-api
+  , cardano-ledger-alonzo-test:plutus-ledger-api
+  , cardano-ledger-babbage:plutus-ledger-api
+  , cardano-ledger-binary:plutus-ledger-api
+  , cardano-ledger-conway:plutus-ledger-api
+  -- Upstream maintainer seems reluctant to fix this.
+  -- https://github.com/phadej/latex-svg/issues/10
+  , latex-svg-image:base
```

Comment:
Notice we are using our own version from CHaP, not the one on Hackage

Reply:
Thanks, I will try updating the one in CHaP.

Reply:
It is a fork of mine (I opened a PR/issue against the upstream one long ago and it is still waiting there). Let me give you permissions for pushing

Reply:
Add a revision for the one in CHaP. PR incoming for https://github.com/jasagredo/latex-svg

Reply:
PR submitted.

Now using the fixed version from CHaP.